arrange_schedule.py

```diff
@@ -1493,9 +1492,9 @@ def main():
                     os._exit(0)
                 else: #Parent
                     alarm_crawler_google_drive_img = raw_time + 3600.0
-            except:
+            except Exception as e:
                 receive_obj["result"] = "fail"
-                receive_obj["error"] = "fork6 error"
+                receive_obj["error"] = "fork6 error" + " : " + str(e)
                 set_system_log(receive_obj)
                 alarm_crawler_google_drive_img = raw_time + 600.0
 
```

Comment:
One commit should do just one thing, if you want to add error message, please create another PR.